src/features/Articles/Articles.tsx

```diff
@@ -0,0 +1,32 @@
+import { ArticleAPI } from '@/api/Article';
+import ArticlesStyle from './Articles.module.scss';
+import Image from 'next/image';
+
+export function Articles({ articles }: { articles: ArticleAPI[] }) {
+  return (
+    <div className={ArticlesStyle.articles}>
+      {articles.map((article) => (
+        <div className={ArticlesStyle.article} key={article.id}>
+          {article.urlToImage ? (
+            <img
```

Comment:
Use `Image` component

Reply:
For this tag it's need to configure universal hostname in my next.config.js, but this feature is not available